apiserver/facades/client/modelconfig/backend.go

```diff
@@ -18,8 +18,6 @@ type Backend interface {
 	common.BlockGetter
 	ControllerTag() names.ControllerTag
 	ModelTag() names.ModelTag
-	ModelConfigValues() (config.ConfigValues, error)
-	UpdateModelConfig(map[string]interface{}, []string, ...state.ValidateConfigFunc) error
 	Sequences() (map[string]int, error)
 	SpaceByName(string) error
```

Comment:
I wonder how much longer we need SpaceByName etc for

Reply:
Done & Done